docs/tables/aws_bedrock_imported_model.md

```diff
@@ -0,0 +1,205 @@
+  custom_model_units
+from
+  aws_bedrock_imported_model;
+```
+
+```sql+sqlite
+select
+  model_name,
+  arn,
+  creation_time,
+  instruct_supported,
+  model_architecture,
+  job_arn,
+  job_name,
+  model_data_source,
+  model_kms_key_arn,
+  custom_model_units
+from
+  aws_bedrock_imported_model;
+```
+
+### List models created in the last 7 days
+Identify recently imported models to track new additions and monitor recent model imports. This helps in maintaining an up-to-date inventory of your models.
+
+```sql+postgres
+select
+  model_name,
+  model_arn,
+  creation_time,
+  instruct_supported,
+  model_architecture
+from
+  aws_bedrock_imported_model
+where
+  creation_time >= now() - interval '7 days';
+```
+
+```sql+sqlite
+select
+  model_name,
+  model_arn,
+  creation_time,
+  instruct_supported,
+  model_architecture
+from
+  aws_bedrock_imported_model
+where
+  creation_time >= datetime('now', '-7 days');
+```
+
+### List models that support instructions
+Identify which imported models have instruction support capabilities. This helps in selecting models suitable for instruction-based tasks.
+
+```sql+postgres
+select
+  model_name,
+  model_arn,
+  creation_time,
+  model_architecture,
+  tags
+from
+  aws_bedrock_imported_model
+where
+  instruct_supported = true;
+```
+
+```sql+sqlite
+select
+  model_name,
+  model_arn,
+  creation_time,
+  model_architecture,
+  tags
+from
+  aws_bedrock_imported_model
+where
+  instruct_supported = 1;
+```
+
+### Get details for a specific model by ARN
+Retrieve comprehensive information about a particular model. This is useful when you need to understand all aspects of a specific imported model.
+
+```sql+postgres
+select
+  model_name,
+  arn,
+  creation_time,
+  instruct_supported,
+  model_architecture,
+  job_arn,
+  job_name,
+  model_data_source,
+  model_kms_key_arn,
+  custom_model_units
+from
+  aws_bedrock_imported_model
+where
+  arn = 'arn:aws:bedrock:us-east-1:123456789012:imported-model/example-model';
+```
+
+```sql+sqlite
+select
+  model_name,
+  arn,
+  creation_time,
+  instruct_supported,
+  model_architecture,
+  job_arn,
+  job_name,
+  model_data_source,
+  model_kms_key_arn,
+  custom_model_units
+from
+  aws_bedrock_imported_model
+where
+  arn = 'arn:aws:bedrock:us-east-1:123456789012:imported-model/example-model';
+```
+
+### List models by architecture type
+Group imported models by their architecture to understand the distribution of different model types in your environment.
+
+```sql+postgres
+select
+  model_architecture,
+  count(*) as model_count,
+  array_agg(model_name) as models
+from
+  aws_bedrock_imported_model
+group by
+  model_architecture
+order by
+  model_count desc;
+```
+
+```sql+sqlite
+select
+  model_architecture,
+  count(*) as model_count,
+  group_concat(model_name, ', ') as models
+from
+  aws_bedrock_imported_model
+group by
+  model_architecture
+order by
+  model_count desc;
+```
+
+### Find models with specific tags
```

Comment:
I couldn’t find the `tags` column in the table schema, so this query might throw an error. Could you please take a moment to validate the query?